src/libtexture/imagecache_pvt.h

```diff
@@ -250,18 +250,27 @@ class OIIO_API ImageCacheFile final : public RefCnt {
     /// Info for each MIP level that isn't in the ImageSpec, or that we
     /// precompute.
     struct LevelInfo {
-        ImageSpec spec;         ///< ImageSpec for the mip level
-        ImageSpec nativespec;   ///< Native ImageSpec for the mip level
+        std::unique_ptr<ImageSpec> m_spec;  ///< ImageSpec for the mip level,
+            // only specified if different from nativespec
+        ImageSpec nativespec;  ///< Native ImageSpec for the mip level
+        mutable std::unique_ptr<float[]> polecolor;  ///< Pole colors
```

Comment:
This is fine for now, but I think that as a further optimization, we can move polecolor to the SubimageInfo struct, where for most files it will be empty/null, but for latlong env maps will point to a single allocation of float[2*nchannels*nmiplevels]. Then we don't even need the per-level pointer.